backend/mediaprovider/subsonic/subsonicmediaprovider.go

```diff
@@ -59,10 +63,12 @@ func (s *subsonicMediaProvider) EditPlaylist(id, name, description string, publi
 }
 
 func (s *subsonicMediaProvider) AddPlaylistTracks(id string, trackIDsToAdd []string) error {
+	s.playlistsCached = nil
```

Comment:
Shouldn't be needed here (or Remove/Update PlaylistTracks) since track info isn't in the *mediaprovider.Playlist type or the playlistsCached slice.

Reply:
Oh wait, number of tracks is. Ignore this :)